### PR TITLE
Navbar was created

### PR DESCRIPTION
**Pull request by vshovkuta**
The component "Navbar" was created. 
For now it contains Logo and two disabled button: 
1. For login - "Sign in"
2. For view the list of shelters - "Find shelter"

In the future, into this component will be aded more buttons/links for work our website.

### CODE REVIEW

File: src/app/navbar/navbar.component.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<nav class="nav mat-elevation-z6">
+  <a mat-button disableRipple [routerLink]="mainLogoHref">
+    <img alt="logo" title="logo" id="logo" [src]="mainLogoSrc">
```
Comment (ksenya-ta) [inline src/app/navbar/navbar.component.html]: do not use id's for styling purposes

Comment (ksenya-ta) [reply src/app/navbar/navbar.component.html]: also alt&title should be something useful for users not developers

File: src/app/navbar/navbar.component.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <span id="title" [innerText]="mainTitleText"></span>
```
Comment (ksenya-ta) [inline src/app/navbar/navbar.component.html]: same here: why do you need this id?

Comment (ksenya-ta) [inline src/app/navbar/navbar.component.html]: do not bind text with innerText, use the regular markup for it: `<span>{{text}}</span>`. Also if this text is supposed to be static don't add variables for it, just write it in the markup.

File: src/app/navbar/navbar.component.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
+/*
+* see https://www.fonts-online.ru/font/Kindly-Jasmine for title font
+* license: free for personal project, non-profit and charity use
+*/
+.nav {
+  display: flex;
+  justify-content: space-between;
+  background-color: aliceblue;
```
Comment (ksenya-ta) [inline src/app/navbar/navbar.component.scss]: move all the colors to variables so everyone can reuse it later

File: src/app/navbar/navbar.component.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  styleUrls: ['./navbar.component.scss']
+})
+export class NavbarComponent implements OnInit {
+
+  constructor() { }
+
+  mainTitle = 'Shelter';
+  mainTitleText = '';
+  mainLogoSrc = '/assets/logo.png';
+  mainLogoHref = [];
```
Comment (ksenya-ta) [inline src/app/navbar/navbar.component.ts]: do not create a variables for static content

File: src/app/navbar/navbar.component.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+  ngOnInit() {
+  }
```
Comment (ksenya-ta) [inline src/app/navbar/navbar.component.ts]: don't keep empty constructor and life-cycle hooks, if you don't need it - remove it.

File: src/app/navbar/navbar.component.html (hunk 1 above)
Comment (vshovkuta) [reply src/app/navbar/navbar.component.html]: The "disableRipple" use to disable material's ripple effect when user click on logo. 
Can I leave this?

Comment (vshovkuta) [reply src/app/navbar/navbar.component.html]: > also alt&title should be something useful for users not developers

I added this for users and screen readers... Remove?

File: src/app/navbar/navbar.component.ts (hunk 4 above)
Comment (vshovkuta) [reply src/app/navbar/navbar.component.ts]: What do you mind? 
Should the 'mainTitleText', 'mainLogoSrc' and 'mainTitle' variables be defined as simple string in HTML-template?
When do another collaborator want to change them, he has to change them in template? I have thought easier change them in the *.ts file than find in HTML-template...

File: src/app/navbar/navbar.component.html (hunk 1 above)
Comment (ksenya-ta) [reply src/app/navbar/navbar.component.html]: do not remove it, just make sure it has a meaningful to users messages

File: src/app/navbar/navbar.component.scss [hunk 6]
```diff
@@ -0,0 +1,32 @@
+/*
+* see https://www.fonts-online.ru/font/Kindly-Jasmine for title font
+* license: free for personal project, non-profit and charity use
+*/
+$background-color: aliceblue;
```
Comment (ksenya-ta) [inline src/app/navbar/navbar.component.scss]: and move it to the separate file with variables you're going to use in all the project files that might need it

**Pull request merged**